src/OpenDiffix.Core.Tests/QueryEngine.Tests.fs

```diff
@@ -226,4 +226,10 @@ type Tests(db: DBFixture) =
       "SELECT count(*) FROM customers_small GROUP BY round_by(age, 10), city ORDER BY 1"
       "SELECT count(*) FROM customers_small GROUP BY city, round_by(age, 10) ORDER BY 1"
 
+  [<Fact>]
+  let ``Duplicated grouping doesn't change results`` () =
+    equivalentQueries
+      "SELECT count(*) FROM customers_small GROUP BY city, city ORDER BY 1"
+      "SELECT count(*) FROM customers_small GROUP BY city ORDER BY 1"
```

Comment:
Will the result also be identical? In postgres I noticed that duplicate labels are dropped.

Reply:
I don't understand the question, can you please rephrase it?
This test asserts that the results are identical.

Reply:
Ah, I see. I thought it just tests for noise layers.